tools/scripts-core/index.js

```diff
@@ -71,7 +71,6 @@ async function runScript() {
 
 switch (command) {
```

Comment:
Maybe we can improve the default case with more info about list of options

Reply:
build-storybook and start-storybook will stay?

Reply:
I would like too. but with the constraint of app and lib should use storybook this is hard